x/oracle/types/assets.go

```diff
@@ -0,0 +1,22 @@
+package types
+
+//nolint
+const (
+	MicroLunaDenom  = "uluna"
+	MicroUSDDenom   = "uusd"
+	MicroKRWDenom   = "ukrw"
+	MicroSDRDenom   = "usdr"
+	MicroCNYDenom   = "ucny"
+	MicroJPYDenom   = "ujpy"
+	MicroEURDenom   = "ueur"
+	MicroGBPDenom   = "ugbp"
+	MicroMNTDenom   = "umnt"

```

Comment:
Remove what we don't need. We probably only need `MicroAtomDenom` and `MicroUmeeDenom`.